rosdep/base.yaml

```
@@ -3133,6 +3133,9 @@ libswscale-dev:
   fedora: [ffmpeg-devel]
   gentoo: [virtual/ffmpeg]
   ubuntu: [libswscale-dev]
+libsystemd-dev:
+  debian: [libsystemd-dev]
+  ubuntu: [libsystemd-dev]
```

```suggestion
  ubuntu:
    artful: [libsystemd-dev]
    bionic: [libsystemd-dev]
    cosmic: [libsystemd-dev]
    xenial: [libsystemd-dev]
```

This package isn't available in Trusty and so can't use a shorthand ubuntu rule. @tfoote is there a more compact representation for this?

---

From entries for other projects, it looks like the format would be:
```suggestion
  ubuntu:
    '*': [libsystemd-dev]
    trusty: null
```
See `xulrunner-1.9.2` or `libglfw3-dev` as examples.

---

It's a balancing act. For adding new rules we sometimes let it slide that we add a generic rule that won't be supported on older platforms. (aka it will incorrectly resolve) Since it's a new key, we know that it won't break anyone on the older system as they cannot already be using it. If they do try to use it on an older system it will fail at install time. Trusty is just on the edge, it's still officially supported but only for another 5 months. But since the wildcard technique is relatively compact i'd suggest going with that as @JWhitleyAStuff suggests. We can then drop the trusty rule in a purge at some later date.

---

Thanks for the feedback. Sounds good!